reference/orders.v2.oas2.yml

```diff
@@ -153,6 +157,14 @@ paths:
                         - id: 230
                           value: '192'
                   customer_locale: en
+                  fees:
+                    - id: 1
+                      type: custom_fee
+                      display_name_customer: ABC Retail Delivery Fee
+                      source: v2
+                      cost_ex_tax: 12.3000
```

Comment:
as we discussed offline, we should add some explanation to clearly call out that the `(cost_ex_tax, cost_inc_tax)` should have the correct values that reflects the tax rate referenced by the `tax_class_id` 

If the `tax_class_id` refers to a 10% tax rate, then the `(cost_ex_tax, cost_inc_tax)` should contain 10% tax when we do the subtraction. 

If the `tax_class_id` is omitted, the store configured "default tax class for fee" would be used and the cost values should match that.

If the user fails to provide accurate data here, downstream operation such as refund may have unintended surprises .